content/02.body.md

```diff
@@ -234,3 +234,13 @@ Alternatively, if degree is not a desired feature, then the edge prior can be us
 
 
 ## Conclusion
+
+Nodes in biomedical networks can have different degrees, and networks can have very different degree distributions.
+Many edge prediction methods are correlated with degree.
+Degree imbalance, in which edges are distributed unequally between nodes, can lead edge prediction methods to make nonspecific or trivial predictions.
+Sometimes networks faithfully represent all true relationships, but degree biases and other misrepresentations can obscure node degree.
```

Comment:
```suggestion
```
I think this paragraph is stronger without this sentence in it. Give it a read without it and see what you think.